misc/python/materialize/mzcompose/services.py

```diff
@@ -970,3 +976,39 @@ def __init__(
                 "volumes": [f"{config.name}:/root/.config/mz/profiles.toml"],
             },
         )
+
+
+class Prometheus(Service):
+    def __init__(self, name: str = "prometheus") -> None:
+        super().__init__(
+            name=name,
+            config={
+                "image": "prom/prometheus:v2.41.0",
+                "ports": ["9090:9090"],
```

Comment:
FWIW I think this is kind of a smell, because it means you can only have one of these workflows running at a time locally. `./mzcompose web grafana` and `./mzcompose web prometheus` are meant to make opening the webface easy even when using ephemeral ports.

Reply:
Ok, will revert to dynamic ports.